frontend/packages/topology/src/filters/KindFilterDropdown.tsx

```diff
@@ -80,7 +80,7 @@ const KindFilterDropdown: React.FC<KindFilterDropdownProps> = ({
           data-test={filter.label}
         >
           <ResourceIcon kind={filter.id} />
-          {t(filter.labelKey)} ({supportedKinds[filter.id]})
+          {filter.labelKey} ({supportedKinds[filter.id]})
```

Comment:
Let us improve this in a follow-up PR. This might confuse other developers in the future.